components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -380,6 +380,9 @@ public enum ErrorMessages {
                 "The shared user profile attributes are read only."),
         ERROR_CODE_ORGANIZATION_OWNER_NOT_EXIST("60096", "The assigned organization owner does not exist ",
                 "The assigned organization owner is not found in the tenant with ID: %s"),
+        ERROR_CODE_ORGANIZATION_NAME_CONTAINS_HTML_CONTENT("60097", "HTML contents are not allowed for " +
+                "organization name.", "Organization name with HTML content is not allowed due to possible XSS " +
+                "attacks."),
```

Comment:
Shall we convey the error message as organization name contained not allowed special characters `<`, `>`, `:` ?

Reply:
The defined regular expression `".*<[^>]+(/>|>.*?</[^>]+>).*"`  only validate HTML like contents. Still characters like `:, ?` can exist.

The regular expression validate embedded HTMLs, 
- Can start and end from any string.
- There should be starting tag `<...>` and ending tag `<../>` or else it can have `<.../>`